doc/RecordingControl.xml

```diff
@@ -2364,6 +2522,109 @@ secfrac = "." 1*6DIGIT</programlisting>
+      <title>Recording segments</title>
+      <para>
+        There are three possible recording strategies as defined for <literal>RecordingTargetConfiguration</literal>:
+        <variablelist>
+          <varlistentry>
+            <term>Cloud (default)</term>
+            <listitem>
+              <para>
+                The device will automatically upload each recorded segment to the Cloud location as soon as possible. After completion or on error the data won't be kept locally on the device. 
+              </para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>Local</term>
+            <listitem>
+              <para>
+                The device will record the segments on its local storage and respect the MaximumRetentionTime of the RecordingConfiguration.
+              </para>
+            </listitem>
+          </varlistentry>
+          <varlistentry>
+            <term>Both</term>
+            <listitem>
+              <para>
+                The device will record the segments on its local storage and respect the MaximumRetentionTime of the RecordingConfiguration. It will also attempt to automatically upload each segment to the Cloud location as they become available.
+              </para>
+            </listitem>
+          </varlistentry>
+        </variablelist>
+      </para>
+      <para>
+        When recording on its local storage, the device may need to delete the oldest recording segments if storage space is insufficient.
+      </para>
+      <para>
+        When uploading to the configured Cloud location, if an error occurs the device can retry but should abandon if the operation took too long and the next segment is ready to be uploaded.
+        Only whole segments should be uploaded and committed to the cloud storage (no incomplete segment should be made available on failure).
+      </para>
+    </section>
+    <section xml:id="_refSegmentExportCommands">
+      <title>Segment export commands</title>
+      <para>
+        The <literal>ListRecordedSegments</literal> command lists the segments currently available for export from the device local storage.
+        For a given time range, this command shall always return the same segments with two exceptions: the newly recorded segments are added as they become
+        available, and the deleted segments are removed from it (for example if the device had to delete them to make room for the new ones).
+      </para>
+      <para>The <literal>ExportRecordedSegments</literal> command requests an export operation for
+        a time-range of segments. A device shall accept multiple request but may queue and process
+        them sequentially. Segments shall be uploaded in chronological order. Progress can be
+        tracked via the AsynchronousOperationStatus event and the method
+          <literal>GetExportRecordedSegmentState</literal>. Operations can be cancelled at any time
```

Comment:
@jflevesque-genetec 
document also needs to be updated, **GetExportRecordedSegmentState** is removed now.